cmd/main.go

```diff
@@ -54,6 +55,15 @@ func main() {
 
 	store.Initialize()
 
+	// Register local OAM traits and workloads
```

Comment:
It might be the case that we'll look to move the "store" capability to MeshKit in the future.